modules/administration-guide/pages/enabling-readiness-init-containers.adoc

```diff
@@ -0,0 +1,6 @@
+[id="enabling-readiness-init-containers"]
```

Comment:
```suggestion
[id="enabling-component-init-containers"]
```

Reply:
These containers calls readiness init containers. Let's keep the term.

Reply:
```suggestion
[id="component-readiness-init-containers"]
```

Reply:
```suggestion
[id="readiness-init-containers"]
```